src/style.css

```diff
@@ -0,0 +1,39 @@
+:root {
+  --star-size: 16px;
+  --header-size: 96px;
+}
+
+* {
+  margin: 0;
+}
+
+.header {
+  width: var(--header-size);
+  height: var(--header-size);
+}
+
+.stars {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  display: inline-block;
```

Comment:
We don't need to change default display. Stars are displayed next to each other thanks to flex parent